src/main/java/SchedularUtilities.java

```diff
@@ -0,0 +1,13 @@
+import java.util.Random;
+
+public class SchedularUtilities {
```

Comment:
Please stop using util or utilites as class or package name.
In this case u can have PriorityGenerator as a class having one public method generate().